src/wallet/rpcwallet.cpp

```diff
@@ -2653,9 +2668,28 @@ Value zc_raw_pour(const json_spirit::Array& params, bool fHelp)
     CDataStream ss(SER_NETWORK, PROTOCOL_VERSION);
     ss << rawTx;
 
+    std::string encryptedBucket1;
+    std::string encryptedBucket2;
+    {
+        CDataStream ss2(SER_NETWORK, PROTOCOL_VERSION);
+        ss2 << ((unsigned char) 0x00);
+        ss2 << pourtx.ephemeralKey;
+        ss2 << pourtx.ciphertexts[0];
+
+        encryptedBucket1 = HexStr(ss2.begin(), ss2.end());
+    }
+    {
+        CDataStream ss2(SER_NETWORK, PROTOCOL_VERSION);
+        ss2 << ((unsigned char) 0x01);
+        ss2 << pourtx.ephemeralKey;
+        ss2 << pourtx.ciphertexts[1];
```

Comment:
This is basically just an ad-hoc "off chain" format for encoding the encrypted notes, to make it fit into our current API, right?